celesta-java/src/main/java/ru/curs/celesta/CelestaProcLogic.java

```diff
@@ -0,0 +1,162 @@
+package ru.curs.celesta;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+class CelestaProcExecutor {
+
+    private final CelestaProcProvider celestaProcProvider;
+    private final Function<JSessionContext, CallContext> callContextProvider;
+
+    public CelestaProcExecutor(CelestaProcProvider celestaProcProvider, Function<JSessionContext, CallContext> callContextProvider) {
```

Comment:
Codacy is now nagging about redundant public modifiers on constructors of CelestaProcExecutor, CelestaProcMeta and CelestaProcProvider classes. And why not fix it?